jsonld-contexts/shellfish-production-areas.jsonld

```diff
@@ -0,0 +1,5 @@
+{
+  "@context": {
+    "ShellfishProductionArea": "https://smartdatamodels.org/ShellfishProductionArea"
```

Comment:
add it in `shellfish-farming.jsonld`?

Reply:
I am not sure I understand. Do you want me to add the relation to shellfish-farming.jsonld for example like this `"ShellfishTable": "https://schema.org/ShellfishTable",` (That is a current example) ?

Reply:
so just move this line to `shellfish-farming.jsonld`

Reply:
Understood